src/Db/Diff/PKSimDB_diff.txt

```diff
@@ -1 +1,48 @@
-INSERT INTO tab_transports(transport_direction,process,source_organ,source_compartment,target_organ,target_compartment,source_organ_type,source_compartment_type,target_organ_type,target_compartment_type) VALUES('InfluxBrainPlasmaToInterstitial','ActiveInfluxSpecificPlasmaToInterstitial','Brain','Plasma','Brain','Interstitial','ORGAN','COMPARTMENT','ORGAN','COMPARTMENT');
+INSERT INTO tab_calculation_method_rates(calculation_method,formula_rate,formula,dimension) VALUES('DiseaseStates','PARAM_Input_GFR','60 * 1E-2 / 1.73','Flow per body surface area');
+INSERT INTO tab_calculation_methods(calculation_method,display_name,description,category,sequence) VALUES('DiseaseStates','Disease states','Calculation of disease states parameters','DiseaseStates',1);
+INSERT INTO tab_categories(category,display_name,description,category_type,icon_name) VALUES('DiseaseStates','Disease States','Disease state parameters','Individual',NULL);
+INSERT INTO tab_container_names(container_type,container_name,display_name,description,extension,sequence,icon_name) VALUES('DISEASE_STATE','CKD','CKD Parameter container','CKD Parameter container',NULL,1,NULL);
+INSERT INTO tab_container_parameter_rates(container_id,container_type,container_name,parameter_name,calculation_method,formula_rate,min_value,min_isallowed,max_value,max_isallowed,value_origin) VALUES(5954,'DISEASE_STATE','CKD','InputGFR','DiseaseStates','PARAM_Input_GFR',0.0,1,0.34683,1,92);

```

Comment:
@Yuri05 USing PVV does not work becuase we have to carry around the specices all the time. So as for Expressions, I am a defining a neew CalculationMethod for DiseaseStates